packages/desktop-client/src/components/settings/Encryption.tsx

```diff
@@ -26,11 +27,13 @@ export function EncryptionSettings() {
 
   return encryptKeyId ? (
     <Setting
-      primaryAction={<Button onPress={onChangeKey}>Generate new key</Button>}
+      primaryAction={
+        <Button onPress={onChangeKey}>{t('Generate new key')}</Button>
+      }
     >
       <Text>
         <Text style={{ color: theme.noticeTextLight, fontWeight: 600 }}>
-          End-to-end Encryption is turned on.
+          {t('End-to-end Encryption is turned on.')}
         </Text>{' '}
         Your data is encrypted with a key that only you have before sending it
```

Comment:
This still needs translated
![image](https://github.com/user-attachments/assets/5f3d80b9-f963-4125-89a9-27a770236b7f)